ex5B/ex5Spec.js

```diff
@@ -0,0 +1,30 @@
+var firstUniqueChar = require('./ex5.js');
+
+describe("function firstUniqueChar", function() {
+	it("finds another unique character e in the string", function() {
+		expect(firstUniqueChar("abcdedcba")).toBe("e");
+	});
+
+	it("return null when we only have one [non-unique] char", function() {
```

Comment:
This title is from an old test. It should say `return null, all chars are duplicated`